src/libengine/BUILD

```diff
@@ -14,10 +14,10 @@
 # limitations under the License.
 #
 
-package(default_visibility = ["//test:__subpackages__"])
```

Comment:
May be easier using an autoformatter for changes like these, but I haven't looked into it too much.

e.g. https://github.com/bazelbuild/buildtools

Probably not worth it to worry about this now if it's a lot of effort though. We can always add something like this when we're done implementing the main functionalities.

Reply:
I've been running `buildifier` when I change Bazel files, but I've only been doing it manually from the command line. I can look into adding a `genrule` that just runs the `buildifier` command on all of the Bazel files when needed.